mmv1/products/compute/api.yaml

```diff
@@ -9750,6 +9750,62 @@ objects:
                     - :NEVER
                     - :ON_PERMANENT_INSTANCE_DELETION
                   default_value: :NEVER
+          - !ruby/object:Api::Type::Map
+            name: 'internalIp'
+            api_name: internalIPs
+            min_version: beta
+            key_name: "interface_name"
```

Comment:
Do we want to add `key_description` here which will add a description for the field `interface_name`  in doc? Current description is "`interface_name` - (Required) The identifier for this object. Format specified above". I wonder if we want to change it.

Reply:
I don't see a clean description on the GCP public doc, perhaps the current description is good enough.